### PR TITLE
Hyper-v : Adding code coverage xmls and scripts

### PR DESCRIPTION
**Pull request by asofron**

### CODE REVIEW

File: WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            <os>Linux</os>
+            <ipv4></ipv4>
+            <sshKey>rhel5_id_rsa.ppk</sshKey>
+            <suite>lisnext</suite>
+            <hardware>
+                <create>true</create>
+                <numCPUs>8</numCPUs>
+                <memSize>8192</memSize>
+                <disableDiff>true</disableDiff>
+                <nic>VMBus,External,001600112313</nic>
```
Comment (chvalean) [inline WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml]: MAC address not required

File: WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        </test>
+    </testCases>
+    <VMs>
+        <vm>
+            <hvServer>localhost</hvServer>
+            <vmName>Build-VHD-gcov</vmName>
+            <os>Linux</os>
+            <ipv4></ipv4>
+            <sshKey>rhel5_id_rsa.ppk</sshKey>
+            <suite>lisnext</suite>
```
Comment (chvalean) [inline WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml]: change this to gcov-kernel - also in above suite definition

File: WS2012R2/lisa/xml/code-coverage/code_cov_create_report.xml [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    <testCases>
+        <test>
+        <testName>install_kernel_gcov</testName>
+        <PreTest>setupScripts\copyLatestItem.ps1</PreTest>
+        <testScript>BuildKernelWithPatches.sh</testScript>
+        <files>remote-scripts/ica/BuildKernelWithPatches.sh,remote-scripts/ica/utils.sh</files>
+        <timeout>3600</timeout>
+        <onError>Abort</onError>
+        <testParams>
+            <param>TC_COVERED=lis-rpms-01</param>
```
Comment (chvalean) [inline WS2012R2/lisa/xml/code-coverage/code_cov_create_report.xml]: TC_COVERED=gcow-kernel-install

File: WS2012R2/lisa/xml/code-coverage/coverage-vss.xml [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+            <hvServer>localhost</hvServer>
+            <vmName>code-coverage-vss</vmName>
+            <os>Linux</os>
+            <ipv4></ipv4>
+            <sshKey>rhel5_id_rsa.ppk</sshKey>
+            <testParams>
+                <param>SSH_PRIVATE_KEY=rhel5_id_rsa</param>
+                <param>snapshotName=ICABase</param>
+                <param>VMSnapshot=ICABase</param>
+                <param>DriveLetter=E:</param>
```
Comment (chvalean) [inline WS2012R2/lisa/xml/code-coverage/coverage-vss.xml]: param no longer used

File: WS2012R2/lisa/xml/code-coverage/coverage-storage.xml [hunk 5]
```diff
@@ -0,0 +1,1243 @@
...
+        <!--<param>vmMemory=8GB</param>-->
+        <!--</testParams>-->
+    </global>
+
+    <testSuites>
+        <suite>
+            <suiteName>storage</suiteName>
+            <suiteTests>
+                <suiteTest>MainVM_Checkpoint</suiteTest>
+                <!-- <suiteTest>VHD_IDE_0_1_DYNAMIC</suiteTest> -->
```
Comment (chvalean) [inline WS2012R2/lisa/xml/code-coverage/coverage-storage.xml]: add new stor tests back here

File: WS2012R2/lisa/xml/code-coverage/coverage-sriov.xml [hunk 6]
```diff
@@ -0,0 +1,785 @@
...
+                <!-- <suiteTest>Move_VHD</suiteTest> -->
+                <suiteTest>Max_vCPU</suiteTest>
+                <suiteTest>One_vCPU</suiteTest>
+                <!-- Multiple VFs tests -->
+                <suiteTest>Multiple_basic</suiteTest>
+                <suiteTest>Multiple_fileCopy</suiteTest>
+                <suiteTest>MaxNICS</suiteTest>
+                <suiteTest>MaxNICS_vCPU</suiteTest>
+                <!-- Pause/Save VM tests with Single/Multiple NICs -->
+                <suiteTest>Single_PauseVM</suiteTest>
```
Comment (chvalean) [inline WS2012R2/lisa/xml/code-coverage/coverage-sriov.xml]: remove Single_PauseVM and Single_SaveVM

File: WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml (hunk 1 above)
Comment (asofron) [reply WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml]: Removed

File: WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml (hunk 2 above)
Comment (asofron) [reply WS2012R2/lisa/xml/code-coverage/build-gcov-vhd.xml]: changed

File: WS2012R2/lisa/xml/code-coverage/code_cov_create_report.xml (hunk 3 above)
Comment (asofron) [reply WS2012R2/lisa/xml/code-coverage/code_cov_create_report.xml]: changed

File: WS2012R2/lisa/xml/code-coverage/coverage-vss.xml (hunk 4 above)
Comment (asofron) [reply WS2012R2/lisa/xml/code-coverage/coverage-vss.xml]: removed

File: WS2012R2/lisa/xml/code-coverage/coverage-storage.xml (hunk 5 above)
Comment (asofron) [reply WS2012R2/lisa/xml/code-coverage/coverage-storage.xml]: Done

File: WS2012R2/lisa/xml/code-coverage/coverage-sriov.xml (hunk 6 above)
Comment (asofron) [reply WS2012R2/lisa/xml/code-coverage/coverage-sriov.xml]: Done

**Pull request merged**